compose.yaml

```diff
@@ -0,0 +1,84 @@
+name: wevote
+
+services:
+  db:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.db
+      tags:
+        - "${COMPOSE_PROJECT_NAME}-db:latest"
+    container_name: "${COMPOSE_PROJECT_NAME}-db"
+    environment:
+      POSTGRES_DB: WeVoteServerDB
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-secret}
+    env_file: .env
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    networks:
+      - ${COMPOSE_PROJECT_NAME}
+    ports:
+      - "5000:5432"
+    restart: unless-stopped
+    user: postgres
+    volumes:
+      - "${COMPOSE_PROJECT_NAME}-postgres-data:/var/lib/postgresql/data"
+    
+  aws:
+    container_name: "${COMPOSE_PROJECT_NAME}-localstack"
+    depends_on:
+      db:
+        condition: service_healthy
+        restart: true
+    environment:
+      DOCKER_HOST: unix:///var/run/docker.sock
+    image: localstack/localstack
+    networks:
+      - ${COMPOSE_PROJECT_NAME}
+    ports:
+      # Gateway
+      - "4566:4566"
+      # External
+      # - "4510-4559:4510-4559"
+    profiles:
+      - optional
+    volumes:
+      - "${COMPOSE_PROJECT_NAME}-localstack:/var/lib/localstack"
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Why do we need to mount the docker socket inside the container?

Reply:
[Mounting the Docker socket `/var/run/docker.sock` as a volume is required for the Lambda service](https://docs.localstack.cloud/getting-started/installation/#starting-localstack-with-docker-compose). [This thread](https://stackoverflow.com/questions/63201603/what-is-the-result-of-mounting-var-run-docker-sock-in-a-docker-in-docker-scen?noredirect=1&lq=1) explains the situation more eloquently than I ever could.

Reply:
Ahh, got it. Thanks for the explanation. I think we're only using localstack for mocking SQS, but in case we use it for more stuff later, this is ready. :)